pkg/actions/toolkit/enable.go

```diff
@@ -0,0 +1,123 @@
+package toolkit
+
+import (
+	"context"
+
+	"github.com/kris-nova/logger"
+	"github.com/pkg/errors"
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/flux"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubeclient "k8s.io/client-go/kubernetes"
+)
+
+const allNamespaces = ""
+
+var defaultFluxComponents = []string{"helm-controller", "kustomize-controller", "notification-controller", "source-controller"}
+
+//go:generate counterfeiter -o fakes/fake_flux_client.go . FluxClient
+type FluxClient interface {
+	PreFlight() error
+	Bootstrap(opts *api.Toolkit) error
+}
+
+type Manager struct {
+	Opts         *api.Toolkit
+	K8sClientSet kubeclient.Interface
```

Comment:
nit: I think we've just used `clientSet` elsewhere, is this just to differentiate `clientSet` and the client for flux? If so how about `KubeClient` to match `FluxClient`?

Reply:
i swear i copied that from somewhere else 🤔 

but yeh i have no great attachment to it